app-backend/datamodel/src/main/scala/com/azavea/rf/datamodel/package.scala

```diff
@@ -85,7 +85,8 @@ package object datamodel {
   }
 
   implicit val crsEncoder: Encoder[CRS] =
-    Encoder.encodeString.contramap[CRS] { crs => s"epsg:${crs.epsgCode}" }
+    Encoder.encodeString.contramap[CRS] { crs => s"epsg:${crs.epsgCode.getOrElse(0)}" }
```

Comment:
I think instead of silence we should give some comments about projection issues.

```scala
crs.epsgCode.getOrElse(throw new Exception("Message here"))
```